### ISSUE TITLE
PDF document has the slide number only in the first page

### ISSUE THREAD

**Issue by a-i-m-a-r** [hakimel/reveal.js#3295]
When I load the presentation with ?print-pdf added, I can see each slide with its slide number on the right bottom corner.
However, when I create the PDF document, the preview only shows the slide number for the first page and the file is created as such. The rest of the pages are unnumbered.
Actually I am interested in the opposite: First page (title) unnumbered and the rest numbered, but I am satisfied if I managed to number all of them.

**Comment by Benson665** [comment]
If am going to receive a every good input of a feedback of good services we both twerk together

**Comment by a-i-m-a-r** [comment]
The problem was that I had established `html.print-pdf .reveal .slides section{top:80px}` in a css file, so the slide content would not overlap the content of the toc-progress plugin, due to `center:false`. Therefore, everything was displaced downward. 
So the page numbers were also displaced. Besides, the background was also displaced which showed a part in the next slide, although in Firefox I did not notice that in Chromium the background color overlapped the toc-progress of the next slide. My first option was to correct the position of the page numbers with `html.print-pdf .reveal .slide-number {bottom:80 px;}`, but finally decided against it. Instead of displacing all the section, I displaced a <div> I inserted gathering the content. That way I did not have to correct the page number position anymore.
Well this works in Chromium. Firefox does not yet print the page numbers. although I see them in the preview.
